config/clusters/carbonplan/cluster.yaml

```diff
@@ -1,5 +1,6 @@
 name: carbonplan
 provider: aws
+account: don't know
```

Comment:
@2i2c-org/tech-team, can someone help with info about the account we currently use to access the carbonplan cluster on aws, please?